.github/workflows/manageiq_cross_repo.yaml

```
@@ -0,0 +1,52 @@
+name: ManageIQ Cross Repo Workflow
+
+on:
+  workflow_call:
+    inputs:
+      test_repo:
+        required: true
+        type: string
+      repos:
+        required: true
+        type: string
+      test_suite:
+        required: false
+        type: string
+
+jobs:
+  ci:
+    name: Run manageiq-cross_repo
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    services:
+      postgres:
+        image: manageiq/postgresql:10
+        env:
+          POSTGRESQL_USER: root
+          POSTGRESQL_PASSWORD: smartvm
+          POSTGRESQL_DATABASE: vmdb_test
+        options: --health-cmd pg_isready --health-interval 2s --health-timeout 5s --health-retries 5
+        ports:
+        - 5432:5432
+    env:
+      TEST_SUITE: ${{ inputs.test_suite }}
+      PGHOST: localhost
+      PGPASSWORD: smartvm
+      REPOS: ${{ inputs.repos }}
+      TEST_REPO: ${{ inputs.test_repo }}
```

Super minor, but let's keep all the inputs together:

```suggestion
      TEST_REPO: ${{ inputs.test_repo }}
      REPOS: ${{ inputs.repos }}
      TEST_SUITE: ${{ inputs.test_suite }}
      PGHOST: localhost
      PGPASSWORD: smartvm

```